packages/web-config-server/src/export/excelFormatters/formatMatrixDataForExcel.js

```diff
@@ -83,6 +88,15 @@ export const formatMatrixDataForExcel = (
     // This table has no row categories, just one set of rows
     formattedData.push(...rows.map(formatRowData));
   }
+
+  // Add export date and origin to the bottom of the sheet
```

Comment:
Is there any way to consolidate this code with the what's in the meditrak-server? Perhaps an 'buildExcelExport' util or something, which receives the data as an array and some metadata, and adds the title and export date and builds the report? What do you think about that?

Reply:
That's a good idea. Will take that.